components/common/logo/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+var curenetics_logo = require('./curenetics_logo_2.png');
+
+const Logo = styled.img`
+  height: 45px;
+  width: 115px;
+  overflow: hidden;
+  top: 20px;
+  left: 50px;
+  position: absolute;
+`;
+
+class Logo extends Component {
+  render() {
+    return(
+    <Wrapper>
+      <Logo><img src={curenetics_logo} alt="Curenetics logo" /></Logo>
```

Comment:
You are wrapping an `<img>` within an `<img>`

1. img is self closing: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
2. this is invalid html, it will look like this:

`<img>
 <img />
</img>
`